_posts/2024-04-29-wait-you-guys-are-writing-unit-tests/wait-you-guys-are-writing-unit-tests.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "Wait, you guys are writing unit tests?"
+description: |
+  A reminder.
+author:
+  - name: Chloe Pou-Prom
+    url: {}
+date: 2024-04-29
+output:
+  distill::distill_article:
+    self_contained: false
+categories:
+    - post-miscellaneous
+draft: true
+preview: unit_tests.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+![](unit_tests.png)
+
+Confession: I rarely write unit tests. 😅 [^1]
```

Comment:
Personally, I like this introduction a lot - it's snappy and catches the attention.